go/vt/sidecardb/schema/twopc/dt_state.sql

```diff
@@ -19,5 +19,6 @@ CREATE TABLE IF NOT EXISTS dt_state
   dtid varbinary(512) NOT NULL,
   state bigint NOT NULL,
   time_created bigint NOT NULL,
-  primary key(dtid)
+  primary key(dtid),
+  key (time_created)
```

Comment:
How many rows do we expect this table to contain?

Reply:
This is user-dependent, we can expect max of 5-10% of transactions per 30 seconds (transaction timeout). 
The transaction watcher keeps querying this table filtering on time with ~1 second frequency, we may also set it less. It would be better to have index on this column.